src/react/components/pages/projectSettings/projectMetrics.tsx

```diff
@@ -0,0 +1,260 @@
+import React from "react";
+import { FormValidation } from "react-jsonschema-form";
+import { AssetState, IAsset, IAssetMetadata, IProject, IRegion } from "../../../../models/applicationState";
+import "vott-react/dist/css/tagsInput.css";
+import _ from "lodash";
+import { AssetService } from "../../../../services/assetService";
+import { strings } from "../../../../common/strings";
+
+/**
+ * Required properties for Project Metrics
+ * @member project - Current project to fill metrics table
+ */
+export interface IProjectMetricsProps {
+    project: IProject;
+}
+
+export interface IProjectMetricsState {
+    loading: boolean;
+    sourceAssets: IAsset[];
+    projectAssetsMetadata: IAssetMetadata[];
+}
+
+/**
+ * @name - Project Form
+ * @description -
+ */
+export default class ProjectMetrics extends React.Component<IProjectMetricsProps, IProjectMetricsState> {
+    public state = {
+        loading: true,
+        sourceAssets: [],
+        projectAssetsMetadata: [],
+    };
+
+    public async componentDidMount() {
+        this.setState({
+            loading: true,
+        });
+
+        await this.getAssetsAndMetadata();
+    }
+
+    public render() {
+        if (this.state.loading) {
+            console.log("Waiting for data");
+            return <h2> Loading...</h2>;
+        }
+
+        const tags = this.props.project.tags || [];
+        const renderTagCategories = tags.map((item) => {
+            return (<li>{item.name}</li>);
+        });
+
+        const tagsMap = this.getTagsCount();
+        // can't map over iterators so this is the workaround
+        // const tagItems = [...tagsMap.entries()];
+        // const renderTagCount = tagItems.map((item) => {
+        //     return (<li><b>{item[0]}:</b> {item[1]} </li>);
+        // });
+
+        const renderTagCount = tags.map((tag) => {
+            const tagName = tag.name;
+            return (
+                <li>
+                    <b>{tagName}: </b>
+                    <span className="tag-count badge badge-light badge-pill">
+                        {tagsMap.get(tagName) || 0}
+                    </span>
+                </li>
+            );
+        });
+
+        return (
+            <div className="project-settings-page-metrics p-3 bg-lighter-1">
+                <h3>
+                    <i className="fas fa-chart-bar fa-1x"/>
+                    <span className="px-2">
+{strings.projectMetrics.title}
+</span>
+                </h3>
+                <ul className="list-group list-group-flush m-3">
+                    <li className="list-group-item total-source-asset">
+                        <b>Total source assets: </b>
+                        <span
+                            className="source-asset-count badge badge-light badge-pill">
```

Comment:
Nit: store `badge badge-light badge-pill` in a variable that is used in each `className`